mysql/charset.go

```diff
@@ -574,15 +574,15 @@ const (
 	UTF8MB4Charset = "utf8mb4"
 	DefaultCharset = UTF8MB4Charset
 	// DefaultCollationID is utf8mb4_bin(46)
-	DefaultCollationID      = 46
-	Latin1CollationID       = 8
-	ASCIICollationID        = 11
-	UTF8CollationID         = 33
-	UTF8MB4CollationID      = 45
-	BinaryCollationID       = 63
-	UTF8DefaultCollation    = "utf8_bin"
-	UTF8MB4DefaultCollation = "utf8mb4_bin"
-	DefaultCollationName    = UTF8MB4DefaultCollation
+	DefaultCollationID        = 46
```

Comment:
Has any risk to change collation id?

Reply:
Here only changes the default collation, IDs of collation are not changed, so there's no risk.

Reply:
BTW, the previous naming is misleading, that's why I renamed those variables.

Reply:
The ID is not changed?

I found collation will be translate to proto, which may be persistent, so it's better not  to change the ID

https://github.com/pingcap/parser/pull/295/files#diff-c31b8654bc3f8f52137403cf3dc617faL724

Reply:
Hi @tiancaiamao The change is the default collation for a given charset, for example, the change of `UTF8MB4CollationID`(before rename) from `45` to `46` means changes default collation of `utf8mb4` from `utf8mb4_general_ci` to `utf8mb4_bin`:

```
-	Latin1CollationID       = 8		+Latin1DefaultCollationID  = 47     // Change default collation of `latin1` from `latin1_swedish_ci` to `latin1_bin`
-	ASCIICollationID        = 11		+ASCIIDefaultCollationID   = 65     // Change the default collation of `ascii` from `ascii_general_ci` to `ascii_bin`
-	UTF8CollationID         = 33		+UTF8DefaultCollationID    = 83     // Change the default collation of `utf8` from `utf8_general_ci` to `utf8_bin`
-	UTF8MB4CollationID      = 45		+UTF8MB4DefaultCollationID = 46     // Change the default collation of `utf8mb4` from `utf8mb4_general_ci` to `utf8mb4_bin`
-	BinaryCollationID       = 63		+BinaryDefaultCollationID  = 63     // The default collation of `binary` is not changed.
```

Reply:
> I found collation will be translate to proto, which may be persistent, so it's better not to change the ID
> 
> https://github.com/pingcap/parser/pull/295/files#diff-c31b8654bc3f8f52137403cf3dc617faL724

`mysql.CollationNames` table is not changed, neither the value of `mysql.BinaryCollationID`.

Again, the old namings are really misleading, PTAL